### PR TITLE
add argument `--metric,-m` and support new metric `wer` (Word Error Rate)

### PR DESCRIPTION
**Pull request by johnklee**
Continuous implementation of #30  & #40

### CODE REVIEW

File: experiments/model_a_eval/constants.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+"""Module to hold constnats."""
```
Comment (wkCircle) [inline experiments/model_a_eval/constants.py:1]: typo: constants

**Review by wkCircle — DISMISSED** [review]
LGTM with only minor typos. Thank you for the support!

File: experiments/model_a_eval/constants.py (hunk 1 above)
Comment (johnklee) [reply experiments/model_a_eval/constants.py]: Fixed. Thanks for the feedback!

**Review by wkCircle — APPROVED** [review]
Thanks for the support!

**Pull request merged**